common/culture/eras/00_culture_eras.txt

```diff
@@ -25,3 +25,8 @@ culture_era_late_medieval = {
 		mercenary_hire_cost_mult = 0.15
 	}
 }
+
+culture_era_early_modern = {
+	year = 1000
```

Comment:
Why you set so late year and dwarves have no techs in this era.

Reply:
What techs are dwarves supposed to have?

Reply:
> What techs are dwarves supposed to have?

At least riflemen lol.

Reply:
None of them are even at late med?  I could suggest that in #65 I add a dwarven riflemen that is in high med, this would account for dwarves having it early, but giving them any tech in Early Modern, the game would require me to give them all the techs of High Med and Late Med

Reply:
> None of them are even at late med? I could suggest that in #65 I add a dwarven riflemen that is in high med, this would account for dwarves having it early, but giving them any tech in Early Modern, the game would require me to give them all the techs of High Med and Late Med

What about gnomes. They're definitely in the last era.